### ISSUE TITLE
Simplify contact picking workflow on iOS.

### ISSUE THREAD

**Issue by albertein** [monday-consulting/ContactChooser#6]
On iOS, when picking a contact, you get to the contact details and only after picking the email the plugin returns the record to js, that workflow is confusing, IMO, it would be way better to return the first email to the plugin client as soon as the contact is selected.

That can be accomplished implementing `peoplePickerNavigationController:shouldContinueAfterSelectingPerson:` instead of `personViewController:shouldPerformDefaultActionForPerson:property:identifier`.

I've got a patch ready for it, but didn't want to send a push request without asking / discussing it first.

**Comment by albertein** [comment]
https://github.com/albertein/ContactChooser/commit/f5266e65cbbf8272cc54980779ef610aa4f8e171

**Comment by malde** [comment]
That'll make it quiet hard to select another email from a list of more than one, would it? (haven't tried it myself)

**Comment by albertein** [comment]
Yeah, you would end picking up the default (or first) email on the list, for my use case that's OK but i can see how it could be an inconvenience. One option would be letting the developer choose the behavior he wants:
`window.plugins.ContactChooser.chooseContact(letUserPickTheEmailOrWhateverWeWantToCallIt, function(contactInfo)`

If this doesn't make sense please let me know, so i can keep a fork with that functionality for my use case.

Thanks!

**Comment by cevatkerim** [comment]
It gets tricky when you want user to choose a phone number as well. I had to modify the plugin for that. But it would be very nice to be able to pass that option.
